shared/login/recover-password/explain-device/index.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import {globalColors} from '../../../styles'
+import {SignupScreen} from '../../../signup/common'
+import {ButtonType} from '../../../common-adapters/button'
+
+export type Props = {
+  deviceName: string
+  deviceType: string
+  onBack: () => void
+  onComplete: () => void
+}
+
+const ExplainDevice = (props: Props) => {
+  const isMobile = props.deviceType === 'mobile'
```

Comment:
This screen always thinks I'm a desktop even when I choose a mobile.  Also can we rename this so we don't collide with the more common `isMobile`.